apriltag_ros/include/apriltag_ros/continuous_detector.h

```diff
@@ -69,6 +74,23 @@ class ContinuousDetector: public nodelet::Nodelet
   void refreshTagParameters();
 
  private:
+  image_geometry::PinholeCameraModel camera_model_;
+  cpr_tf2_ros::CameraFrameTransformer camera_frame_transformer_;
```

Comment:
since we only care about the position and not the orientation, I'd honestly not bother using this and just transfrom the target pose to optical frame with the tf listener directly. It would also remove the dependency on internal code in here.